test/run/__snapshots__/version-check.spec.ts.snap

```diff
@@ -1,10 +1,10 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`init hook first run through displays correct message when the installed version not meeting project 1`] = `
+exports[`VersionCheck run first run through displays correct message when the installed version not meeting project 1`] = `
 [
   [
     "WARNING: this version: 0.9.23 does not meet the project requirements: >=1.0.0 <2.0.0
-
+        
```

Comment:
very minor, but there seem to be spaces/tabs now in this file

Reply:
Fixed 👍🏼